src/Groups/GAPGroups.jl

```diff
@@ -435,6 +435,35 @@ julia> length(small_generating_set(abelian_group(PermGroup, [2,3,4])))
    return res
 end
 
+"""
+    minimal_generating_set(G::GAPGroup)
```

Comment:
I am a bit confused by the name and the docstring of this function. If it does not guarantee to return a minimal generating set, why not call it `small_generating_set`? That function already exists for different types.

Reply:
That was a copy&paste mistake. Fixed in #2816; I hope that one will be merged quickly then I'll rebase this PR.

For this PR I still need to adjust some tests, working on that.

Reply:
(And thanking you for catching this)